### PR TITLE
Added LunarLander-v2 env with dqn notebook

### PR DESCRIPTION
**Pull request by nishantkr18**

### CODE REVIEW

File: q_learning/lunar_lander_dqn.ipynb [hunk 1]
```diff
@@ -0,0 +1,549 @@
...
+    "size_t consecutiveEpisodes = 50;"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "// Function to train the agent on mlpack's own implementation of the CartPole environment.\n",
```
Comment (zoq) [inline q_learning/lunar_lander_dqn.ipynb:205]: Hm, we don't train on the mlpack implementation.

Comment (nishantkr18) [reply q_learning/lunar_lander_dqn.ipynb]: Right! I always tend to forget this change :sweat_smile:

**Comment by nishantkr18** [comment]
> Is there anything else you like to add or should we merge this one?

Nothing from my side. I guess we can merge this now.

**Comment by nishantkr18** [comment]
Oops, forgot a small url change. With that, I guess its ready :)

**Pull request merged**